index.js

```diff
@@ -104,9 +109,18 @@ prog
     const errArray = []
     const globOpts = { nodir: true }
     const acornOpts = { ecmaVersion: e, silent: true }
-    if (esmodule) { acornOpts.sourceType = 'module'}
 
-    logger.debug(`ES-Check: Going to check files using version ${e}${esmodule ? ' with esmodule set' : ''}`)
+    logger.debug(`ES-Check: Going to check files using version ${e}`)
+
+    if (esmodule) {
```

Comment:
Since we now report more than one option in verbose mode. I don't think it makes sense to do it all on the same line.